.github/workflows/test.yaml

```diff
@@ -1,13 +1,18 @@
 name: test
 on:
   pull_request:
+    types: [opened, reopened, synchronize]
   push:
     branches:
       - main
 jobs:
   tests:
     runs-on: ubuntu-latest
     timeout-minutes: 10
+    permissions:
+      contents: read
+      actions: read
+      pull-requests: write
```

Comment:
I would add a comment that those permissions are required for `fgrosse/go-coverage-report` actions, like this https://github.com/neondatabase/autoscaling/blob/5c838f1509330d3bdb074178928950d4d56ec521/.github/workflows/approved-for-ci-run.yml#L101-L103